diff-containers/test/Test/Data/Map/Diff/Strict.hs

```diff
@@ -43,123 +46,219 @@ import           Test.Util
 -- small values often.
 tests :: TestTree
 tests = testGroup "Data.Map.Diff.Strict" [
-      localOption (QuickCheckTests 1000) $
-      testGroupWithProxy (Proxy @(DiffHistory (OftenSmall Int))) [
+      localOption (QuickCheckTests 10000) $
+      testGroupWithProxy (Proxy @(NormalDiffHistory (OftenSmall Int))) [
           testSemigroupLaws
         , testMonoidLaws
         , testGroupLaws
         ]
     , localOption (QuickCheckTests 1000) $
-      testGroupWithProxy (Proxy @(Diff (OftenSmall Int) (OftenSmall Int))) [
+      testGroupWithProxy (Proxy @(NormalDiff (OftenSmall Int) (OftenSmall Int))) [
           testSemigroupLaws
         , testMonoidLaws
         , testGroupLaws
         ]
-    , localOption (QuickCheckTests 10000) $
-      testProperty "prop_diffingIsPositive" $
-        prop_diffingIsPositive @(OftenSmall Int) @(OftenSmall Int)
-    , localOption (QuickCheckTests 10000) $
-      testProperty "prop_diffThenApply" $
-        prop_diffThenApply @(OftenSmall Int) @(OftenSmall Int)
-    , localOption (QuickCheckTests 10000) $
-      testProperty "prop_applyMempty" $
-        prop_applyMempty @(OftenSmall Int) @(OftenSmall Int)
-    , localOption (QuickCheckMaxRatio 100) $
-      localOption (QuickCheckTests 1000) $
-      testProperty "prop_applyAllAndApplySum" $
-        prop_applyAllAndApplySum @(OftenSmall Int) @(OftenSmall Int)
-    , localOption (QuickCheckMaxRatio 100) $
-      localOption (QuickCheckTests 1000) $
-      testProperty "prop_unsafeApplyAllAndUnsafeApplySum" $
-        prop_unsafeApplyAllAndUnsafeApplySum @(OftenSmall Int) @(OftenSmall Int)
+    , testGroup "diffing" [
+          localOption (QuickCheckTests 10000) $
+          testProperty "prop_diffingIsPositive" $
+            prop_diffingIsPositive @(OftenSmall Int) @(OftenSmall Int)
+        , localOption (QuickCheckTests 10000) $
+          testProperty "prop_diffingIsNormal" $
+            prop_diffingIsNormal @(OftenSmall Int) @(OftenSmall Int)
+        ]
+    , testGroup "Preserving positivity and normality" [
+          localOption (QuickCheckTests 1000) $
+          testProperty "prop_summingDiffsPreservesPositivity" $
+            prop_summingDiffsPreservesPositivity @(OftenSmall Int) @(OftenSmall Int)
+        , localOption (QuickCheckTests 10000) $
+          testProperty "prop_summingDiffHistoriesPreservesPositivity" $
+            prop_summingDiffHistoriesPreservesPositivity @(OftenSmall Int)
+        , localOption (QuickCheckTests 10000) $
+          testProperty "prop_summingDiffsPreservesNormality" $
+            prop_summingDiffsPreservesNormality @(OftenSmall Int) @(OftenSmall Int)
+        , localOption (QuickCheckTests 100000) $
```

Comment:
Same question here? Is this running more than the others for some reason?

Reply:
It's a less expensive test, so I run it more. 

In testing `diff-containers`, I experienced that running too few tests would often not hit interesting test cases that ended in test failure. I've tried to counter this by making the generators generate test cases from a smaller set (see `OftenSmall`), but even then I did not want to make the generators too specific, so I run more tests